### DISCUSSION TITLE
The top GitHub projects per country

### DISCUSSION THREAD

**Discussion by adeonir** [frontendbr/forum#534] [Dicas]
Olá pessoal, tudo certo?

Vi esse link num grupo de slack e achei bem interessante compartilhar aqui:
https://medium.com/@hoffa/the-top-github-projects-per-country-92c275e19409

![1 kofu0xfjoocybpuva-ypwa](https://cloud.githubusercontent.com/assets/3428733/24905426/57eb9e78-1e8a-11e7-9a73-479619e93dd1.png)

**Comment by gabrieldarezzo** [comment]
Caramba que legal, parabéns a todos envolvidos do fórum.


Ps:
Interessante esse ranking, não conhecia o [serenata-de-amor](https://github.com/datasciencebr/serenata-de-amor) por exemplo.

**Comment by felipefialho** [comment]
@frontendbr/admins 😄

**Comment by diogomoretti** [comment]
Detalhe: a maioria desses repo (90%) não são de código :sweat:

**Comment by devigor** [comment]
@diogomoretti é uma pena mesmo
Temos que mudar essa situação
